### PR TITLE
Update Auth0 integration and LoginMethodsSnippet

### PR DESCRIPTION
**Pull request by nmoskaleva**
- update steps for Auth0 integration
- add `acr_values` for BankID Biometrics

### CODE REVIEW

File: src/pages/verify/integrations/auth0.mdx [hunk 1]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
 ### Create the OIDC connection(s)
 You create an OIDC connection for every login method you intend to use.
 
-1. Go to Auth0 dashboard for your tenant and under **Connections** chose **Enterprise**.
+1. Go to Auth0 dashboard for your tenant and under **Authentication** chose **Enterprise**.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:69]: chose -> choose

File: src/pages/verify/integrations/auth0.mdx [hunk 2]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
...
+3. Enter **Connection name**.
 4. Under **Issuer URL** enter the login-method specific URL, as described above.
-5. Choose _Front Channel_ or _Back Channel_ as a **Type**, depending on how you intend to integrate it with you application.
-  -  you should choose _Front Channel_ only if you are developing pure SPA application. In that case, make sure to enable _Callback on location hash_ in Criipto application management, instead  of OAuth2 Code Flow. Otherwise choose _Back Channel_.
-6. Under **Client ID** enter **Client ID/Realm** from your Criipto Verify application.
-7. If you chose _Back Channel_ as a **Type**, enter **Client Secret** generated by Criipto Verify when you enabled the OAuth2 Code Flow.
-8. Copy the **Callback URL** generated by Auth0 and enter it under **Callback URLs** in Criipto Verify application management.
-9. Select **Save changes**.
-10. Make sure to enable created connection for your Auth0 application.
+5. Under **Client ID** enter **Client ID/Realm** from your Criipto Verify application.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:73]: enter -> enter the

File: src/pages/verify/integrations/auth0.mdx [hunk 3]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
...
+6. Copy the **Callback URL** generated by Auth0 and enter it under **Callback URLs** in Criipto Verify application management.
+7. Back in Auth0 dashboard, create connection by clicking **Create**.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:75]: in -> in the

and 

connection -> a connection

File: src/pages/verify/integrations/auth0.mdx [hunk 4]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
...
+8. After the connection is created, go the **Settings** tab. 
+9. Choose _Front Channel_ or _Back Channel_ as a **Type**, depending on how you intend to integrate it with your application.
+  -  you should choose _Front Channel_ only if you are developing pure SPA application. Otherwise choose _Back Channel_.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:78]: pure -> a pure

File: src/pages/verify/integrations/auth0.mdx [hunk 5]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
...
+10. If you chose _Back Channel_ as a **Type**, enter **Client Secret** generated by Criipto Verify when you enabled the OAuth2 Code Flow.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:79]: enter -> enter the

File: src/pages/verify/integrations/auth0.mdx [hunk 6]
```diff
@@ -66,17 +66,19 @@ import LoginMethodsSnippet from '../../../snippets/login-methods-and-path-encode
...
+11. Select **Save Changes**.
+12. Make sure to enable created connection for your Auth0 application.
```
Comment (sgryt) [inline src/pages/verify/integrations/auth0.mdx:81]: created -> the created

**Review by sgryt — COMMENTED** [review]
Minor suggestions - may be incorrect, mostly based on gut feeling.

**Comment by nmoskaleva** [comment]
Thanks @sgryt, all updated.

**Pull request merged**